### PR TITLE
Ref #13270 - fix(query.twig): Inline events removed

### LINKED ISSUE #13270: Remove inline javascript events
See more information in previous issue:
#12261 (Remove inline javascript)

Below remained places that use inline JS events:
```
./view_operations.php:        <td><input type="text" size="20" name="new_name" onfocus="this.select()"
./view_create.php:        . ' onfocus="this.select()" maxlength="64"'
./view_create.php:    . ' onfocus="this.select()"'
./view_create.php:    $htmlString .= ' onclick="selectContent(this, sql_box_locked, true)"';
./templates/columns_definitions/column_definitions_form.phtml:                               onfocus="this.select()" />
./templates/database/designer/having_query_panel.twig:                                    onclick="document.getElementById('query_having').style.display = 'none';" />
./templates/database/designer/aggregate_query_panel.twig:                                    onclick="document.getElementById('query_Aggregate').style.display = 'none';" />
./templates/database/designer/side_menu.phtml:           onclick="build_query('SQL Query on Database', 0)"
./templates/database/designer/rename_to_panel.twig:                                    onclick="document.getElementById('query_rename_to').style.display = 'none';" />
./templates/database/designer/where_query_panel.twig:                                    onclick="document.getElementById('query_where').style.display = 'none'" />
./templates/database/structure/check_all_tables.phtml:         / <a href="#" onclick="unMarkAllRows('tablesForm'); <?= $overhead_check; ?> return false;"><?= __('Check tables having overhead'); ?></a>
./templates/server/databases/table_row.phtml:        <a onclick="PMA_commonActions.setDb('<?= PMA\libraries\Sanitize::jsFormat($current['SCHEMA_NAME']) ?>');"
./templates/table/index_form.phtml:      onsubmit="if (typeof(this.elements['index[Key_name]'].disabled) != 'undefined') this.elements['index[Key_name]'].disabled = false;">
./templates/table/index_form.phtml:                       onfocus="this.select()" />
./templates/table/index_form.phtml:                                   onfocus="this.select()"name="index[columns][sub_parts][]"
./templates/table/index_form.phtml:                                   onfocus="this.select()"
./templates/table/structure/actions_in_table_structure.phtml:            <a href="#" onclick="$('input:checkbox').prop('checked', false);
./templates/table/structure/actions_in_table_structure.phtml:            <a href="#" onclick="$('input:checkbox').prop('checked', false);
./templates/table/structure/add_column.phtml:<form method="post" action="tbl_addfield.php" id="addColumns" name="addColumns" onsubmit="return checkFormElementInRange(this, 'num_fields', '<?= str_replace
./templates/table/structure/add_column.phtml:    <?php $num_fields = '<input type="number" name="num_fields" value="1" onfocus="this.select()" min="1" required />'; ?>
./templates/table/structure/add_column.phtml:    <select name="after_field" onchange="checkFirst()">
./templates/table/search/column_comparison_operators.phtml:        onchange="changeValueFieldType(this, <?= $search_index; ?>)">
./templates/table/search/zoom_result_form.phtml:                        <a href="#" onclick="displayHelp();">
./libraries/sql.lib.php:            . ' onsubmit="return ! emptyCheckTheField(this,'
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'newname\')">';
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'newname\')">';
./libraries/operations.lib.php:        . ' onsubmit="return emptyCheckTheField(this, \'new_name\')">'
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'new_name\')">'
./libraries/sql_query_form.lib.php:        $auto_sel = ' onclick="selectContent(this, sql_box_locked, true);"';
./libraries/sql_query_form.lib.php:            . 'multiple="multiple" ondblclick="insertValueQuery()">';
./libraries/sql_query_form.lib.php:                . ' value="&lt;&lt;" onclick="insertValueQuery()"'
./libraries/sql_query_form.lib.php:                . ' onclick="insertValueQuery()" />';
./libraries/display_change_password.lib.php:        . 'onclick="pma_pw.value = \'\'; pma_pw2.value = \'\'; '
./libraries/display_change_password.lib.php:        . 'onclick="document.getElementById(\'text_pma_change_pw\').focus();" '
./libraries/tracking.lib.php:        . '<option value="execution" onclick="alert(\''
./libraries/plugin_interface.lib.php:            $ret .= ' onclick="if (!this.checked &amp;&amp; '
./libraries/server_status_processes.lib.php:            $retval .= 'onmouseout="$(\'.soimg\').toggle()" '
./libraries/server_status_processes.lib.php:                . 'onmouseover="$(\'.soimg\').toggle()"';
./libraries/rte/rte_footer.lib.php:        $retval .= "onclick='$.datepicker.initialized = false;'>";
./libraries/rte/rte_routines.lib.php:    $retval .= "       class='rte_form ajax' onsubmit='return false'>\n";
./libraries/replication_gui.lib.php:    $html .= ' onsubmit="return checkAddUser(this);">';
./libraries/replication_gui.lib.php:        . '" onclick="suggestPassword(this.form)" />'
./libraries/display_export.lib.php:    $multi_values .= ' onclick="setSelectOptions'
./libraries/display_export.lib.php:    $multi_values .= ' onclick="setSelectOptions'
./libraries/display_export.lib.php:            . '" id="buttonGo" onclick="check_time_out('
./libraries/display_export.lib.php:    $html .= '" onfocus="this.select()" />';
./libraries/display_export.lib.php:    $html .= '" size="5" onfocus="this.select()" />';
./libraries/plugins/auth/AuthenticationCookie.php:                echo ' onchange="document.forms[\'login_form\'].'
./libraries/plugins/transformations/abs/DateFormatTransformationsPlugin.php:            return '<dfn onclick="alert(\'' . Sanitize::jsFormat($source, false) . '\');" title="'
./libraries/DbSearch.php:            . 'onclick="setSelectOptions(\'db_search\','
./libraries/DbSearch.php:            . 'onclick="setSelectOptions(\'db_search\','
./libraries/normalization.lib.php:        . '" onclick="goToFinish1NF();"'
./libraries/normalization.lib.php:        . '" onclick="goToStep4();"'
./libraries/server_privileges.lib.php:       . ' onsubmit="return checkAddUser(this);"'
./libraries/server_privileges.lib.php:        . 'onsubmit="return checkAddUser(this);" '
./libraries/DisplayResults.php:            . 'onsubmit="return '
./libraries/DisplayResults.php:        $onsubmit = 'onsubmit="return '
./js/functions.js:    value="' + PMA_messages.strGenerate + '" onclick="suggestPassword(this.form)" /><input type="text" name="generated_pw" id="generated_pw" /></td></tr>');
./js/functions.js:    <span class="options"><input type="button" class="button" id="button_generate_password" value="' + PMA_messages.strGenerate + '" onclick="suggestPassword(this.form)" /></span>
./js/functions.js:                    PMA_messages.pickColumn + '" onclick="autoPopulate(\'' + colid + '\',' + i + ')"/></td>';
./js/functions.js:                                PMA_messages.pickColumn + '" onclick="autoPopulate(\'' + colid + '\',' + i + ')"/></td>';
./js/pmd/history.js:                str += '<img src="' + pmaThemeImage + 'pmd/or_icon.png" onclick="and_or(' + i + ')" title="OR"/></td>';
./js/pmd/history.js:                str += '<img src="' + pmaThemeImage + 'pmd/and_icon.png" onclick="and_or(' + i + ')" title="AND"/></td>';
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_delete(' + i + ')>' + PMA_getImage('b_drop.png', 
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_edit(' + i + ')>' + PMA_getImage('b_edit.png', 
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_delete(' + i + ')>' + PMA_getImage('b_drop.png', 
./js/normalization.js:                $('.tblFooters').html('<input type="button" onClick="processDependencies(\'\', true);" value="' + PMA_messages.strDone + '"/>');
./js/normalization.js:                $('.tblFooters').html('<input type="submit" value="' + PMA_messages.strDone + '" onclick="processDependencies(\'' + escapeJsString(escapeHtml(data.primary_key)) + 
./js/normalization.js:                        "onclick=\"$('#newCols').html('');$(this).parent().html('')\"/>");
./js/normalization.js:                        "onclick=\"$('#newCols').html('');$(this).parent().html('')\"/>");
./js/normalization.js:            $('.tblFooters').html('<input type="submit" value="' + PMA_messages.strCancel + '" onclick="$(\'#newCols\').html(\'\');$(\'#extra input[type=checkbox]\').prop(\'checked
./js/normalization.js:                '<input type="submit" value="' + PMA_messages.strGo + '" onclick="moveRepeatingGroup(\'' + escapeJsString(escapeHtml(repeatingCols)) + '\')"/>');
```

### PR DESCRIPTION
**Pull request by Seezly**
Signed-off-by: Sergio Gutierrez <sergiogutierrez0611@gmail.com>

### Description

Added event handlers for some inline events.

Related to #13270

Before submitting a pull request, please review the following checklist:

- [X] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [X] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [X] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [X] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: templates/sql/query.twig [hunk 1]
```diff
@@ -18,7 +18,7 @@
             <div class="col">
               <div class="mb-3">
                 <textarea class="form-control" tabindex="100" name="sql_query" id="sqlquery" cols="{{ textarea_cols }}" rows="{{ textarea_rows }}"
-                  {{- textarea_auto_select ? ' onclick="Functions.selectContent(this, sqlBoxLocked, true);"' }} aria-label="{% trans 'SQL query' %}">
+                  {{- textarea_auto_select }} aria-label="{% trans 'SQL query' %}">
```
Comment (williamdes) [inline templates/sql/query.twig:21]: The condition should be kept, maybe as a `data-` attribute

Comment (Seezly) [reply templates/sql/query.twig:21]: Thanks!

Should I write the whole `onclick="Functions.selectContent(this, sqlBoxLocked, true);"` inside the `data-` attribute?

Comment (williamdes) [reply templates/sql/query.twig:21]: I would suggest storing the `textarea_auto_select` boolean in an attribute and using the value of the attribute in the JS listener you added

**Comment by williamdes** [comment]
Hi @Seezly 
I am sorry for the branch mistake, that closed the PR..
Anyway now that your master branch is clean by mistake, would you mind creating a new branch and applying this patch ?
```patch
commit 15e095d0f0b6436ce59b4888a504eaa8b32e8d23 (HEAD -> mm)
Author: Sergio Gutierrez <sergiogutierrez0611@gmail.com>
Date:   Sun Mar 28 12:00:35 2021 -0400

    Ref #13270 - Inline events removed from query.twig
    
    Signed-off-by: Sergio Gutierrez <sergiogutierrez0611@gmail.com>

diff --git a/js/src/sql.js b/js/src/sql.js
index 5442d52b4f..24f7ecd7b2 100644
--- a/js/src/sql.js
+++ b/js/src/sql.js
@@ -216,6 +216,7 @@ AJAX.registerTeardown('sql.js', function () {
     $('body').off('click', 'form[name="resultsForm"].ajax button[name="submit_mult"], form[name="resultsForm"].ajax input[name="submit_mult"]');
     $(document).off('submit', '.maxRowsForm');
     $(document).off('click', '#view_as');
+    $(document).off('click', '#sqlquery');
 });
 
 /**
@@ -867,6 +868,13 @@ AJAX.registerOnload('sql.js', function () {
     $('#view_as').on('click', function () {
         Functions.selectContent(this, sqlBoxLocked, true);
     });
+
+    $('#sqlquery').on('click', function () {
+        // Jquery keeps the attribute value to a boolean 
+        if ($(this).data('textarea-auto-select') === true) {
+            Functions.selectContent(this, sqlBoxLocked, true);
+        }
+    });
 }); // end $()
 
 /**
diff --git a/templates/sql/query.twig b/templates/sql/query.twig
index fb1be01296..65eed8702e 100644
--- a/templates/sql/query.twig
+++ b/templates/sql/query.twig
@@ -17,8 +17,7 @@
           <div class="row">
             <div class="col">
               <div class="mb-3">
-                <textarea class="form-control" tabindex="100" name="sql_query" id="sqlquery" cols="{{ textarea_cols }}" rows="{{ textarea_rows }}"
-                  {{- textarea_auto_select ? ' onclick="Functions.selectContent(this, sqlBoxLocked, true);"' }} aria-label="{% trans 'SQL query' %}">
+                <textarea class="form-control" tabindex="100" name="sql_query" id="sqlquery" cols="{{ textarea_cols }}" rows="{{ textarea_rows }}" data-textarea-auto-select="{{ textarea_auto_select ? "true" : "false"}}" aria-label="{% trans 'SQL query' %}">
                   {{- query -}}
                 </textarea>
               </div>
```

**Comment by Seezly** [comment]
@williamdes Do I have to make another PR?

**Comment by williamdes** [comment]
> @williamdes Do I have to make another PR?

Yes, sorry for that
Do not forget to create a new branch from master

**Pull request closed without merging**